malpolon/models/standard_prediction_systems.py

```diff
@@ -160,14 +163,18 @@ def _step(
         y_hat = self(x)
 
         loss = self.loss(y_hat, self._cast_type_to_loss(y))  # Shape mismatch for binary: need to 'y = y.unsqueeze(1)' (or use .reshape(2)) to cast from [2] to [2,1] and cast y to float with .float()
-        self.log(f"loss/{split}", loss, **log_kwargs)
+        self.log(f"loss_{split}", loss, **log_kwargs)
```

Comment:
The `/` (slash) character was chosen as a separator between the quantity to monitor and the split, because Tensorboard recognizes it and gathers all quantity-related measurements in a single section. This is no the case with other characters.
Hence, `/` (slash) should be kept as a separator.

![Screenshot 2024-11-08 at 20-03-48 TensorBoard](https://github.com/user-attachments/assets/05c227d7-d45b-4568-8d32-41bc6b6fd887)
![Screenshot 2024-11-08 at 20-03-31 TensorBoard](https://github.com/user-attachments/assets/0b5b59d8-2167-4b15-b85c-428e28630afd)
![Screenshot 2024-11-08 at 20-03-19 TensorBoard](https://github.com/user-attachments/assets/f37ae09e-d39b-4f9e-a344-56ebb0897c12)

Reply:
Changing that character would also mean updating every example

Reply:
I didn't think of that. I undid it